src/tests/system/docs/config.rst

```diff
@@ -0,0 +1,338 @@
+Multihost configuration
+#######################
+
+The multihost configuration file contains definition of the domains, hosts and
+their roles that are available to run the tests. It uses the `YAML
+<https://en.wikipedia.org/wiki/YAML>`__ language.
+
+Basic definition
+****************
+
+.. code-block:: yaml
+
+    domains:
+    - type: <domain type>
```

Comment:
There is missing domain name.

Reply:
Name was removed.

Reply:
That simplification is nice until we work with forests where we have multiple domains of the ad type in metadata.
So far I saw the framework crashing with multiple domains of the same type.

```
 - name: domain.com
    type: ad
    hosts:
      - name: ad1
        role: ad
        netbios: DOMAIN
        group: ad_root
        domain_level: top
        os: win-2022-latest
  - name: child.domain.com
    type: ad
    parent: domain.com
    hosts:
      - name: ad2
        role: ad
        netbios: CHILD
        group: ad_subdomain
        os: win-2022-latest
        domain_level: sub1
  - name: tdomain.com
    type: ad
    parent: domain.com
    hosts:
      - name: ad3
        role: ad
        netbios: TDOMAIN
        group: ad_treedomain
        os: win-2022-latest
        domain_level: tree1
```

Reply:
As for multiple domains, I tested with a patch (https://github.com/neoave/mrack/pull/238)  to mrack and managed to get a few more topologies matched so maybe it is no longer a major roadblock. I am not sure about how it would work with ad forests and playbooks that might expect multiple domains.